langchain/vectorstores/base.py

```diff
@@ -9,6 +9,10 @@
 class VectorStore(ABC):
     """Interface for vector stores."""
 
+    @abstractmethod
+    def add_texts(self, texts: List[str]) -> None:
```

Comment:
Perhaps we could replace the type signatures to use `Iterable` over `List` so the code doesn't assume that the text is loaded in memory / can fit in memory?

Reply:
lets do it!